src/config.rs

```diff
@@ -103,6 +103,21 @@ pub enum MessageFilterAction {
         content: String,
         requires_armed: bool,
     },
+    /// Ban the user who sent the offending piece of content.
+    Ban {
+        reason: String,
+        delete_message_seconds: u32,
+    },
+    /// Kick the user who sent the offending piece of content.
+    Kick {
+        reason: String,
+    },
+    /// Timeout the user who sent the offending piece of content.
+    Timeout {
+        reason: String,
+        /// How long to mute the user for, in seconds.
+        duration: i64,
```

Comment:
This should probably be a `u64` (or, more reasonably, a `u32` due to the limits of floating point representations, which this will eventually be transmuted to) unless we need to encode a sign bit. What's the type that the Twilight API needs?

Reply:
The twilight API requires i64

Reply:
Ergh. This is fine enough then